pom.xml

```diff
@@ -281,6 +281,8 @@ limitations under the License.
 
     <!-- Libraries version that MUST be used in all children project -->
 
+    <current-year>2017</current-year>
+    <rudder-major-version>4.1</rudder-major-version>
     <rudder-version>4.1.0~beta4-SNAPSHOT</rudder-version>
```

Comment:
does that works:
`      <rudder-version>${rudder-major-version}.0~beta4-SNAPSHOT</rudder-version> `

And if yes, do we want it to be like that?